### ISSUE TITLE
Show in the toolbar how the underlines look instead of just the name of the underline.

### ISSUE THREAD

**Issue by andresinaka** [andresinaka/Transformer#3]
It would be great to actually show how the underlines look in the toolbar instead of just showing the name.

<img width="135" alt="screen shot 2017-11-30 at 10 34 47" src="https://user-images.githubusercontent.com/1957233/33423826-3f187848-d5ba-11e7-8d09-23b0edd31d21.png">

**Comment by marion59000** [comment]
Hi @andresinaka, I would like to work on this issue, is it possible?

**Comment by andresinaka** [comment]
yes! be my guest! @marion59000 it's yours :)

**Comment by marion59000** [comment]
@andresinaka, I'm working on this issue but I met a problem

I saw in the code that quill create an html element ql-picker-options with the different select ql-picker-item .
I don't know where in the code to add the style like "style-single" to the ql-picker-item class to display the correct underline.

**Comment by andresinaka** [comment]
mmm I've been checking how I would do it and It's not that straightforward...

I tried some things, for example if you go to main.css and add this:

```
.ql-picker-label[data-value="style-thick"]::before {
  text-decoration: none;
  border-bottom: 3px solid black;
}
```
Check the selector i'm using there... it's pretty cool, I didn't even know you can select using a data attribute!

`ql-picker-label` is the class applied to the name of the selected style.
`ql-picker-item` is the class applied to the items of a style in a dropdown, so if you do the same 

When you select the ".style-thick" in the editor you will get something like this:
![image](https://user-images.githubusercontent.com/1957233/33713110-33b978cc-db4a-11e7-8432-cb004a7ca136.png)

BTW, it looks pretty ugly when we add the style to the style selection label haha. 

Let me know how it goes!

**Comment by marion59000** [comment]
Thanks for the tip, I will try it this week-end.

**Comment by andresinaka** [comment]
fixed #8
